README.md

```diff
@@ -918,6 +918,23 @@ Options for the [OpenId Connect](http://developer.okta.com/docs/api/resources/oi
       authScheme: 'OAUTH2'
     }
     ```
+- **customButtons** - Adds an array of custom Auth buttons configurable by the properties below:
```

Comment:
This shouldn't be in the OIDC section, should it? I.e. it could be part of socialAuth, but you could also have a custom button outside of that flow right? Probably just create a new section (like helpLinks before this).

Also, in the style of helplinks, should reword to something like:

You can add custom buttons underneath the login form on the primary auth page by setting the following config option. If you'd like to change the divider text, use the `i18n` config option.